app/assets/json/translations/react/tl.json

```
@@ -1136,30 +1168,30 @@
   "t.applicationForListing": "Aplikasyon para sa %{listingName}",
   "t.applicationSubmitted": "Naisumite na ang Aplikasyon",
   "t.applicationsClosed": "Sarado na ang mga Aplikasyon",
+  "t.areYouSureYouWantToLeave": "Sigurado ba kayong gusto ninyong umalis sa page na ito?  Mawawala sa inyo ang anumang hindi nai-save na impormasyon na nailagay na ninyo sa aplikasyon para sa nakalistang pabahay na ito. Kung gusto ninyong i-save ang inyong impormasyon para tapusin ang aplikasyon sa ibang panahon, paki-click ang a ‘I-save  at Tapusin sa ibang panahon (Save and Finish later)’ na button.",
   "t.areYouSureYouWantToLeaveConfirmation": "Sigurado ba kayong gusto ninyong umalis sa page na ito? Kapag ni-link ninyo ang inyong aplikasyon sa isang account, magiging mas madali ang pagtingin sa estado ng inyong aplikasyon at pagkumpleto sa mga aplikasyon sa hinaharap.",
   "t.areYouSureYouWantToLeaveSignIn": "Siguaro ba kayong gusto ninyong umalis sa page na ito? Kapag gumawa kayo ng account, magiging mas madali ang pagtingin sa estado ng inyong aplikasyon at pagkumpleto sa mga aplikasyon sa hinaharap.",
-  "t.areYouSureYouWantToLeave": "Sigurado ba kayong gusto ninyong umalis sa page na ito?  Mawawala sa inyo ang anumang hindi nai-save na impormasyon na nailagay na ninyo sa aplikasyon para sa nakalistang pabahay na ito. Kung gusto ninyong i-save ang inyong impormasyon para tapusin ang aplikasyon sa ibang panahon, paki-click ang a ‘I-save  at Tapusin sa ibang panahon (Save and Finish later)’ na button.",
   "t.at": "sa",
   "t.available": "puwedeng makuha",
   "t.back": "Bumalik sa nakaraang page",
   "t.child": "anak",
   "t.children": "mga bata",
   "t.close": "Isara na",
   "t.confirm": "Kumpirmahin",
-  "t.continueWithYourApplication": "Gusto ba ninyong ipagpatuloy ang inyong aplikasyon?",
   "t.continue": "Magpatuloy",
+  "t.continueWithYourApplication": "Gusto ba ninyong ipagpatuloy ang inyong aplikasyon?",
   "t.dahliaSanFranciscoHousingPortal": "DAHLIA San Francisco Housing Portal",
-  "t.deleteApplication": "Gusto ba ninyong i-delete ang aplikasyong ito? ",
   "t.delete": "I-delete",
+  "t.deleteApplication": "Gusto ba ninyong i-delete ang aplikasyong ito? ",
```

again, not certain about these whitespaces but they aren't in other languages 🤔 

```suggestion
  "t.deleteApplication": "Gusto ba ninyong i-delete ang aplikasyong ito?",
```

---

I noticed for some of these, the english value had a preceding/trailing space and so I think they were just honoring that.